src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -538,7 +538,10 @@ def transform_manifest(self, format, filters=None, request_config_file='request_
         if not filters:
             filters = {"file": {}}
         filters = filters['file']
-        manifest_config = request_config['manifest']
+        if format == 'bdbag':
+            manifest_config = request_config['bdbag']
```

Comment:
```suggestion
            manifest_config = request_config['bdbag' if format == 'bdbag' else 'manifest']
```

would have been more concise